grc/utils/flask_child_form_add_custom_errors.py

```diff
@@ -1,9 +1,11 @@
 from flask_wtf import FlaskForm
 from wtforms import FieldList
+from grc.utils.logger import LogLevel, Logger
 
+logger = Logger()
 
 def add_error_for_child_form(parent_form_field_list: FieldList, child_form: FlaskForm, child_form_field_name: str, error_message: str):
-    print(f"field_list.errors: {parent_form_field_list.errors}", flush=True)
+    logger.log(LogLevel.WARN, f"field_list.errors: {parent_form_field_list.errors}")

```

Comment:
I think these can just be removed completely.

Reply:
The print statements as well?